android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -259,6 +261,12 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
             }
         };
         gestureDetector = new GestureDetectorCompat(themedReactContext, gestureListener);
+
+        String[] languages = Locale.getISOLanguages();
+        for (String language : languages) {
+            Locale locale = new Locale(language);
+            this.localeMap.put(locale.getISO3Language(), locale);
```

Comment:
you don't have to use `this.` for localeMap and other instance variables that are referenced in this file. `this` would only be required in case you have a local variable that has the same name as an instance variable, in that case, you use `this.` to indicate that you want to access the instance variable not the local/scope one.